src/ExampleException.java

```diff
@@ -1,7 +1,7 @@
 public class ExampleException {
   public static int rectangleArea(int a, int b) {
-    // TODO put your code here
-    return 0;
+      if (a > 0 && b > 0) return a * b;
```

Comment:
it should pass with zero values according to the task, only negative values should fail